ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -410,6 +413,13 @@ struct vk_device_struct {
     vk_pipeline pipeline_flash_attn_f32_f16_D128_cm2[GGML_TYPE_COUNT][2][2][2];
     vk_pipeline pipeline_flash_attn_f32_f16_D256_cm2[GGML_TYPE_COUNT][2][2][2];
 
+    vk_pipeline pipeline_flash_attn_f32_f16_D64_coopmat[GGML_TYPE_COUNT][2][2][2];
```

Comment:
```suggestion
    vk_pipeline pipeline_flash_attn_f32_f16_D64_cm[GGML_TYPE_COUNT][2][2][2];
```
or `_cm1` would be more consistant with the `_cm2` above.

Reply:
done, thanks